s3_to_redshift.go

```diff
@@ -113,6 +114,12 @@ func runCopy(db *redshift.Redshift, inputConf s3filepath.S3File, inputTable reds
 	return nil
 }
 
+func init() {
```

Comment:
why set this in an init function? I'd suggest doing it in the same place as the other args, after flag.parse